label_sync/README.md

```diff
@@ -0,0 +1,50 @@
+# label_sync
+
+Update or migrate github labels on repos in a github org based on a YAML file
+
+## Configuration
+
+A typical labels.yaml file looks like:
+
+```yaml
+---
+labels:
+  - color: 00ff00
+    name: lgtm
+  - color: ff0000
+    name: priority/P0
+    previously:
+      - color: 0000ff
+        name: P0
+  - name: dead-label
+    color: cccccc
+    deleteAfter: 2017-01-01T13:00:00Z
+```
+
+This will ensure that:
+
+- there is a green `lgtm` label
+- there is a red `priority/P0` label, and previous labels should be migrated to
+  it:
+  - if a `P0` label exists:
+    - if `priority/P0` does not, modify the existing `P0` label
+    - if `priority/P0` exists, `P0` labels will be deleted, `priority/P0` labels
+      will be added
+- if there is a `dead-label` label, it will be deleted after
+  2017-01-01T13:00:00Z
+
+## Usage
+
+```sh
+# add or migrate labels on all repos in the knative org
+# Under kubernetes/test-infra/label_sync, run:
+bazel run //label_sync -- \
```

Comment:
I thought we don't use bazel

Reply:
It is pointing to `kubernetes/test-infra`. Normally users should not run this command directly as they won't have such a privileged github account, so this is mainly for demo purpose.
If you don't like it I can delete it though.

Reply:
I guess this entire README was copy/pasted, how about just provide a link to k8s/test-infra README to avoid diverge

Reply:
It's not entirely copied/pasted, e.g. the Deployment part is different. I think the `Configuration` part is worth to be here to make it clear on how to configure it, but I agree this `Usage` is not that useful.
Changed it to point to kubernetes/test-infra.